packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -12,13 +12,12 @@ function DefaultAnswersComponent({ isLoading, hits = [] }) {
   );
 }
 
-export default function Answers({
+function useAnswers({
   searchClient,
   queryLanguages,
   attributesForPrediction,
-  nbHits = 1,
-  answersComponent: AnswersComponent = DefaultAnswersComponent,
-  ...extraParameters
+  nbHits,
+  extraParameters,
```

Comment:
I don't think we should expose "extraParameters", especially considering how it's `extraParameters.params` for search options to be passed. Let's spread here too 👍

Reply:
Ah you're right! https://github.com/algolia/react-instantsearch/pull/2996/commits/cd530682856d8ffd29db202a61fe83f52710ee05